src/core/middleware/security.js

```diff
@@ -37,7 +37,7 @@ export function csp({ _config = config, noScriptStyles, _log = log } = {}) {
     _config.get('CSP') !== 'false' ? deepcopy(_config.get('CSP')) : false;
 
   if (cspConfig) {
-    if (noScriptStyles) {
+    if (noScriptStyles && !_config.get('isDevelopment')) {
```

Comment:
I'm ambivalent about using `!isDevelopment` here because it obscures the fact that the local server is relying on `unsafe-inline`. Maybe one day the local server will not.

I thought of suggesting `!config.get('allowUnsafeInline')` to make it more explicit but I also like how `isDevelopment` is something that would never leak into production.

What do you think?

Reply:
I also thought about introducing a new config variable but as you wrote already, too many config vars may lead to vars leaking into the wrong environment.

It seems slightly safer to me to use `isDevelopment`.

Reply:
I have added a `log.debug` call when we omit the hash-source (and therefore allow `unsafe-inline`).